src/components/sections/get-in-touch-form.svelte

```diff
@@ -0,0 +1,98 @@
+<section id="Get-In-Touch-Section">
+  <h2>Get in touch</h2>
+  <p>
+    Have any questions for us? Want to work with us? Just want to say hi?
+    Whatever it is, feel free to contact us! Send us an email by filling out the
+    form below, and we’ll have a representative get back to you as soon as
+    possible. Don’t worry, we’re friendly! 😊
+  </p>
+  <form id="Contact-Form" action="#">
+    <formfield>
+      <lable for="form-name"></lable>
```

Comment:
In the instances where you use the `<lable>` tag, update the typo to become `<label>` instead.